tlsconfig/tlsconfig_client.go

```diff
@@ -72,3 +72,10 @@ func ClientRootCAs(certPoolProvider CertPoolProvider) ClientParam {
 func ClientCipherSuites(cipherSuites ...uint16) ClientParam {
 	return clientParam(cipherSuitesParam(cipherSuites...))
 }
+
+func InsecureSkipVerify() ClientParam {
```

Comment:
Can you update this to follow the `Client` naming convention -> `ClientInsecureSkipVerify()`.
Additionally can you add a doc comment that includes the default value if not set.

Reply:
Done